platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/aibot/LoadingAnimation.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package foundry.intellij.compose.aibot
+
+import androidx.compose.animation.core.Animatable
+import androidx.compose.animation.core.LinearOutSlowInEasing
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.keyframes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.delay
+import org.jetbrains.jewel.foundation.theme.JewelTheme
+
+// tutorial from https://www.youtube.com/watch?v=xakNOVaYLAg
+
+@Composable
```

Comment:
nit: let's make this a doc on the function rather than this floating comment, that way it's attached in kdoc

Reply:
Is this okay? 843a85b025b0f564952bcdfd208f11ef2b1bc8bc

Reply:
Yeah but let's add a proper description to it too, make the source just a footnote at the end for reference

Reply:
Does this description look good? 23bf68de2e6691756662189c46170cd1210e3b4a

Reply:
Not quite. The kdoc should say what the composable itself does, not just where it's from. Let's do both

```kotlin
/**
 * This shows a loading indicator of three bouncing dots, etc etc.
 *
 * @see <a href="https://...">Source</a>
 */
```

Reply:
Sounds good, thanks! d6d709817002cbbd112499aa1a6a1ea5d6bb73ba 
I made the adjustment here.

Reply:
Adjusted here: b457fec214ebbdbfcb43f01ab5de439336f40b6c